frontend/packages/dev-console/src/components/builds/BuildsTabListPage.tsx

```diff
@@ -0,0 +1,122 @@
+import { useK8sModel } from '@console/shared/src/hooks/useK8sModel';
+import NamespacedPage, { NamespacedPageVariants } from '../NamespacedPage';
+import CreateProjectListPage from '../projects/CreateProjectListPage';
+
+interface ServingListPageProps {
+  match: Rmatch<{ ns?: string }>;
+}
+
+const useResourceListPages = () => {
+  const resourceListPageExtensions = useExtensions<ResourceListPage>(isResourceListPage);
+  const dynamicResourceListPageExtensions = useExtensions<DynamicResourceListPage>(
+    isDynamicResourceListPage,
+  );
+  return React.useMemo(
+    () => getResourceListPages(resourceListPageExtensions, dynamicResourceListPageExtensions),
+    [resourceListPageExtensions, dynamicResourceListPageExtensions],
+  );
+};
+
+const BuildsTabListPage: React.FC<ServingListPageProps> = ({ match }) => {
+  const namespace = match.params.ns;
+
+  const { t } = useTranslation();
+
+  const resourceListPages = useResourceListPages();
+
+  const title = t('devconsole~Builds');
+
+  const menuActions: MenuActions = {};
+  const pages: Page[] = [];
+
+  const extraProps = {
+    showTitle: false,
+    canCreate: false,
+    match,
+  };
+
+  // BuildConfigs
+  const buildConfigLoader = resourceListPages.get('build.openshift.io~v1~BuildConfig');
+  if (buildConfigLoader) {
+    menuActions.buildConfig = {
+      label: t('devconsole~BuildConfig'),
+      onSelection: () => `/k8s/ns/${namespace}/buildconfigs/~new`,
+    };
+    pages.push({
+      href: '',
+      name: t('devconsole~BuildConfigs'),
+      component: () => <AsyncComponent loader={buildConfigLoader} {...extraProps} />,
+    });
+  }
+
+  // We might add a generic extension for multi tab list pages in the future.
+  // Currently we just check if Shipwright is installed and load a (dynamic) resource list page.
+  const shipwrightBuildSupported = useFlag('SHIPWRIGHT_BUILD');
+  const shipwrightBuildLoader = resourceListPages.get('shipwright.io~v1alpha1~Build');
+  const [shipwrightBuildModel] = useK8sModel('shipwright.io~v1alpha1~Build');
+  const shipwrightBadge = shipwrightBuildModel && getBadgeFromType(shipwrightBuildModel.badge);
+  if (shipwrightBuildSupported && shipwrightBuildLoader) {
+    menuActions.shipwrightBuild = {
+      label: t('devconsole~Shipwright Build'),
+      onSelection: () => `/k8s/ns/${namespace}/shipwright.io~v1alpha1~Build/~new`,
+    };
+    pages.push({
+      href: 'shipwright-builds',
+      name: t('devconsole~Shipwright Builds'),
+      component: () => (
+        <AsyncComponent loader={shipwrightBuildLoader} {...extraProps} badge={shipwrightBadge} />
+      ),
+    });
+  }
+
+  if (!match.params.ns) {
+    return (
+      <CreateProjectListPage title={title}>
+        {(openProjectModal) => (
+          <Trans t={t} ns="devconsole">
+            Select a Project to view the list of builds or{' '}
+            <Button isInline variant="link" onClick={openProjectModal}>
+              create a Project
```

Comment:
```suggestion
              Create a Project
```

Reply:
Used the same text from https://github.com/openshift/console/blob/release-4.10/frontend/packages/dev-console/src/components/buildconfig/BuildConfigPage.tsx#L35 and just changed `list of BuildConfigs` to `list of builds` as this can show the other Builds now as well.